registries.c

```diff
@@ -19,6 +19,26 @@ GPtrArray* tmp_values;
 char * headers [] = { "registries", "insecure_registries", "block_registries" };
 gchar *cur_header = "None";
 
+/*
+ * Check if registries are still being exported from
+ * sysconfig
+ */
+void check_envs(){
+	char* env_variables[] = {
+			"ADD_REGISTRY",
+			"INSECURE_REGISTRY",
+			"BLOCK_REGISTRY"
+	};
```

Comment:
I think you could make this simpler by adding a 0 to the end of the list.  
for (char *p=env_variables; p; p++ {
}